include/model.php

```diff
@@ -274,6 +277,7 @@ public function get_language( $value ) {
 				foreach ( $lang->get_tax_props( 'term_id' ) as $term_id ) {
 					$this->cache->set( 'language:' . $term_id, $lang );
 				}
+				$this->cache->set( 'language:tt:' . $lang->get_tax_prop( 'language', 'term_taxonomy_id' ), $lang );
```

Comment:
Maybe we could move this inside the loop above.

Reply:
I didn't think it was useful but if you think it is, let's move it.
On 2nd thought, either it is useful or not, I think we should do that for consistency.
Done in [107e031](https://github.com/polylang/polylang/pull/1250/commits/107e031398ab2e49225e88fb61acd7afad8d6201).